src/chttpd/src/chttpd_misc.erl

```diff
@@ -294,11 +296,25 @@ dbs_info_callback({error, Reason}, #vacc{resp = Resp0} = Acc) ->
 
 handle_task_status_req(#httpd{method='GET'}=Req) ->
     ok = chttpd:verify_is_server_admin(Req),
-    {Replies, _BadNodes} = gen_server:multi_call(couch_task_status, all),
-    Response = lists:flatmap(fun({Node, Tasks}) ->
-        [{[{node,Node} | Task]} || Task <- Tasks]
-    end, Replies),
-    send_json(Req, lists:sort(Response));
+    Jobs = lists:foldl(fun(Type, JobsAcc) ->
+        NewJobs = couch_jobs_fdb:get_active_jobs(Type),
+        maps:merge(NewJobs, JobsAcc)
+    end, #{}, ?ACTIVE_TASK_JOB_TYPES),
+    ActiveTasks = maps:map(fun(JobId, _) ->
+        fabric2_fdb:transactional(fun(Tx) ->
```

Comment:
We could move most of this to a fabric2, could be in fabric2_active_tasks or similar though we could also make it smaller if say we get the list of active jobs and their data in the same transaction say one per type. Then we know the data is there and don't have to check for `{error, _}`:

```
    maps:map(fun(Type) ->
        couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(undefined), fun(JTx) ->
            JobIds = couch_jobs:get_active_jobs(JTx, Type),
            lists:foldl(fun(TasksAcc) ->
               {ok, Data} = couch_jobs:get_job_data(JTx, Type, JobId),
               maps:merge(maps:get(<<"active_tasks_info">>, Data, #{}), TasksAcc)
           end, #{}, JobIds)
    end, ?ACTIVE_TASK_JOB_TYPES)